src/projects.rs

```diff
@@ -101,6 +106,23 @@ impl ProjectsClient {
         Ok(project)
     }
 
+    pub async fn delete(&self, id: String) -> Result<(), Error> {
+        let url = format!("{}/v2/projects/{}", self.client.address, id);
+        self.client.req(Method::DELETE, &url, None).send().await?;
+        Ok(())
+    }
+
+    pub async fn list(
+        &self,
+        _: Option<ProjectsSelector>,
```

Comment:
The projects selector is currently empty and unimplemented on the server side, but exists to preserve the possibility of future expansion without having to change client function signatures.